db/fixtures/chargeback_rates.yml

```diff
@@ -17,6 +17,18 @@
           :finish: Infinity
           :fixed_rate: 0.0
           :variable_rate: 0.02
+    - :description: Used CPU Cores
+      :group: cpu
+      :source: used
+      :metric: cpu_usage_rate_average
```

Comment:
@amaurygonzalez here. For VMs the metrics collector fills the ```cpu_usagemhz_rate_average```
for container projects the metrics collector fills ```cpu_usage_rate_average```